packages/manager-react-components/src/components/Modal/Modal.component.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  OdsButton,
+  OdsModal,
+  OdsText,
+  OdsSpinner,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_MODAL_COLOR,
+  ODS_BUTTON_COLOR,
+  ODS_TEXT_PRESET,
+  ODS_SPINNER_SIZE,
+} from '@ovhcloud/ods-components';
+
+export enum ModalTypeEnum {
+  'info' = 'info',
```

Comment:
let's keep the enum items UPPERCASED to match code style ?

Reply:
It's fixed